apps/native-component-list/src/screens/AV/AudioPlayer.tsx

```diff
@@ -128,7 +152,74 @@ export default class AudioPlayer extends React.Component<Props, State> {
         setRateAsync={this._setRateAsync}
         setIsMutedAsync={this._setIsMutedAsync}
         setVolume={this._setVolumeAsync}
+        extraIndicator={<JsiAudioBar isPlaying={this.state.isPlaying} sound={this._sound!} />}
       />
     );
   }
 }
+
+// for some reason, iOS returns much smaller sample values
+const inputRange = Platform.OS === 'ios' ? [0, 0.3] : [0, 1];
+
+function JsiAudioBar({ sound, isPlaying }: { sound: Audio.Sound; isPlaying: boolean }) {
```

Comment:
Extract this component to the separate file and optimise it as much as possible by:
- getting rid of any inlined `style={{ }}` by either using `StyleSheet.create({ ... })` or some `const computedStyles = useMemo(() => ({ ...stylesToBeComputed }, [...computedStylesDeps])`